core/src/main/java/org/verapdf/processor/plugins/Attribute.java

```diff
@@ -2,16 +2,16 @@
  * This file is part of veraPDF Library core, a module of the veraPDF project.
  * Copyright (c) 2015-2024, veraPDF Consortium <info@verapdf.org>
  * All rights reserved.
- *
+ * <p>
```

Comment:
Please remove these <p>

Reply:
this <p> is required here

Reply:
![pic](https://github.com/user-attachments/assets/21d41ed0-484c-409c-9f59-64d4600954b4)